includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -150,18 +150,27 @@ public function getFooter() {
 	}
 
 	public function getNavigation() {
-		global $wgUser;
 		$this->checkRequestCompleteness();
 
 		// TODO: change to not mocked data
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)